src/materials/MeshPhongMaterial.js

```diff
@@ -11,6 +11,8 @@
  *
  *  map: new THREE.Texture( <Image> ),
  *
+ *  energyConverving: false
```

Comment:
I would prefer that you not implement changes to the models here. That is not the purpose of this PR. We should all have the opportunity to discuss and test proposed changes in a separate PR.